csharp/src/Drivers/Apache/ApacheParameters.cs

```diff
@@ -25,5 +25,15 @@ public class ApacheParameters
         public const string PollTimeMilliseconds = "adbc.apache.statement.polltime_ms";
         public const string BatchSize = "adbc.apache.statement.batch_size";
         public const string QueryTimeoutSeconds = "adbc.apache.statement.query_timeout_s";
+        public const string IsMetadataCommand = "adbc.apache.statement.is_metadata_command";
+
+        public const string CatalogName = "adbc.apache.catalog_name";
+        public const string SchemaName = "adbc.apache.schema_name";
+        public const string TableName = "adbc.apache.table_name";
+        public const string TableTypes = "adbc.apache.table_types";
+        public const string ColumnName = "adbc.apache.column_name";
+        public const string ForeignCatalogName = "adbc.apache.foreign_catalog_name";
+        public const string ForeignSchemaName = "adbc.apache.foreign_schema_name";
+        public const string ForeignTableName = "adbc.apache.foreign_table_name";
```

Comment:
I wish I had insisted more strongly on a different set of names in the beginning because now we're never going to change them :(. I find it's "just weird" to call these things "adbc.apache" as if the entire project wasn't an Apache project already. They should have been named something like "adbc.hiveserver2".

Anyway, I think we should take a cue from the existing option names for ingestion e.g. `adbc.ingest.target_catalog`  and name these after their semantic purpose, so maybe something like `adbc.get_metadata.target_catalog`, `adbc.get_metadata.target_db_schema`, etc. trying to aim for maximum consistency with metadata identifier names that already exist.

Reply:
@CurtHagenlocher 
Using the pattern `adbc.get_metadata.*`. Let me know if these names make sense.